newdle/client/src/components/ParticipantTable.js

```diff
@@ -75,6 +72,36 @@ function AvailabilityRow({
   const duration = useSelector(getNewdleDuration);
   const startTime = toMoment(startDt, 'YYYY-MM-DDTHH:mm');
 
+  const isMobile = useIsMobile();
+  const renderParticipants = () => {
+    if (!numberOfParticipants) {
+      return 'There are no participants yet.';
+    }
+    if (participants.length === 0) {
+      return 'Nobody has voted yet.';
+    }
+    if (availableCount === 0) {
+      return 'Nobody is available.';
+    }
```

Comment:
This isn't a good idea because it prevents you from seeing *who* "nobody" actually is.

Reply:
Why does it prevent you from seeing that? You can see it in the message underneath, right?

I added this mainly because there was an awkward empty white space under the grey area (of the circle + date). Now that everything is in one color it can also be removed.

Reply:
But this is for the case where you have named participants, they are just all "red".

Reply:
Ah, true. I mixed it up with "Nobody has voted yet"